pino.js

```diff
@@ -306,9 +352,3 @@ module.exports.stdSerializers = {
   res: serializers.asResValue,
   err: serializers.asErrValue
 }
-module.exports.pretty = pretty
-Object.defineProperty(
-  module.exports,
-  'LOG_VERSION',
-  {value: LOG_VERSION, enumerable: true}
-)
```

Comment:
Why have you removed these? I know that the log version is defined on the `pinoPrototype` so that removal could be a 👍, but I don't know if there is a use case for it being on the module (@davidmarkclements @mcollina ?).

The export of `pretty`, though, needs to remain. Did I miss that export in your changes? Or is it actually removed?